src/registrar/tests/test_models_domain.py

```diff
@@ -544,9 +582,28 @@ def test_user_adds_one_nameserver(self):
                 to the registry
             And `domain.is_active` returns False
         """
-        raise
 
-    @skip("not implemented yet")
+        # set 1 nameserver
+        nameserver = "ns1.my-nameserver.com"
+        self.domain.nameservers = [(nameserver,)]
+
+        # when you create a host, you also have to update at same time
```

Comment:
**comment**: This comment confused me because I didn't realize that these commands were being set up to use in testing with `check_call`. Maybe "when we create a hose, we should have updated at the same time"?